src/copilot/chatCommands.ts

```diff
@@ -38,12 +56,42 @@ export interface CommandDefinition {
 
 export const CHAT_COMMANDS: Record<string, CommandDefinition> = {
     // Simple command shortcuts - these are handled directly and don't go to language model
+    help: {
+        type: CommandType.Simple,
+        requiresConnection: false,
+        skipConnectionLabels: true, // Provides its own connection status
+        handler: async (stream, controller, connectionUri) => {
+            stream.markdown(`${loc.helpWelcome}\n\n`);
+            stream.markdown(`**${loc.helpWhatICanDo}**\n\n`);
```

Comment:
Based on the screenshots, I think this would look better with just one `\n` afterward (and the colon mentioned in the other comment).